### DISCUSSION TITLE
Trying to run a 16 bit  cli tool on windows 11

### DISCUSSION THREAD

**Discussion by Kartikey-star** [otya128/winevdm#1328] [General]
I am trying to run a cli tool on windows 11. The tool happens to be of 16 bit and does not work on windows 11. I followed the steps listed in the doc and created a shortcut on desktop for otvdm and adding the exe file as well. After that when i run the application it does not open.What i want to achieve is to use the 16 bit cli tool on the 64 bit OS. Any ideas?

**Comment by cracyc** [comment]
Win16 never had a command line and 16bit windows programs couldn't output to it.  It's almost certainly a dos program so try dosbox or msdos-player.

**Reply by Kartikey-star** [reply]
Thanks. It seems i was making an error while using the tool. Closing the thread.
